react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,33 @@
+//import logo from './logo.svg';
+import './App.css';
+import './ToDoItem.css';
```

Comment:
Minor adjustment but .css files are usually done kebab-case style

Reply:
done with that